vdp/pipeline/v1beta/connector_definition.proto

```diff
@@ -95,12 +99,21 @@ message ConnectorDefinition {
   // Connector definition custom flag, i.e., whether this is a custom
   // connector definition.
   bool custom = 11 [(google.api.field_behavior) = OUTPUT_ONLY];
-  // Connector definition icon URL.
+  // Connector definition icon URL. This is a public URL that allows clients to
+  // download and render the connector icon.
   string icon_url = 12 [(google.api.field_behavior) = OUTPUT_ONLY];
   // Connector definition vendor name.
   string vendor = 13 [(google.api.field_behavior) = OUTPUT_ONLY];
   // Vendor-specific attributes.
   google.protobuf.Struct vendor_attributes = 14 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Source code URL. This points to the source code where the connector is
+  // implemented.
+  string source_url = 15 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Connector definition version. This is a string that fulfills the SemVer
+  // specification (e.g. `1.0.0-beta`).
+  string version = 16 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
ℹ️ SemVer should be enough to determine both the version and the release stage (if it includes the `-beta` or `-alpha` suffix). For now this should be enough for the frontend, I need to verify that we could also filter by the release stage applying a filter on this field.